src/constants/screens.js

```diff
@@ -1,43 +1,60 @@
 /**
- * This file cannot be converted to TypeScript because it's used by the Tailwind config
+ * This file cannot be converted to TypeScript because it's referenced in the
+ * Tailwind config.
+ *
+ * @typedef {'2xl' | 'xl' | 'lg' | 'md' | 'sm' | 'xs'} Breakpoint
+ * @typedef {Exclude<Breakpoint, 'xs'>} RealBreakpoint
```

Comment:
It would be nice to explain why `xs` is not a `real` breakpoint. Or find a clearer name here.

Reply:
Added a comment to explain `RealBreakpoint` vs `Breakpoint` in 2c4dae0.